terraform/modules/cloudfront/distribution/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "origin" {
+  type = object({
+    lb_dns_name   = string
+    lb_http_port  = number
+    lb_https_port = number
+  })
+}
+
+# variable "iam_certificate_id" {
+#   type = string
+# }
```

Comment:
Do we need this?

Reply:
This is a placeholder for when I point the DNS to the distribution. I think each distribution will need to have the cert that its corresponding LB currently has.